src/activities/categorization/resource/board/category_flowers.qml

```diff
@@ -0,0 +1,116 @@
+/* GCompris
+ *
+ * Copyright (C) 2017 Aman Kumar Gupta <gupta2140@gmail.com>
```

Comment:
2017 -> 2018

Reply:
I have to update this PR after image dataset is merged into gcompris-words repo. So, i'll make all the changes at once :)